### PR TITLE
add alert toast on credential success

### PR DESCRIPTION
**Pull request by jamestalton**
Signed-off-by: James Talton <jtalton@redhat.com>

### CODE REVIEW

File: frontend/src/routes/Credentials/CredentialsForm.tsx [hunk 1]
```diff
@@ -949,10 +951,20 @@ export function CredentialsForm(props: {
...
+                    })
                     history.push(NavigationPath.credentials)
                 })
             } else {
                 return createResource(stateToData() as IResource).promise.then(() => {
+                    toastContext.addAlert({
+                        title: t('credentialsForm.created.title'),
+                        type: 'success',
+                        autoClose: true,
+                    })
```
Comment (showeimer) [inline frontend/src/routes/Credentials/CredentialsForm.tsx:967]: A small enhancement might be to add which Credential was actually created/updated?

Comment (jamestalton) [reply frontend/src/routes/Credentials/CredentialsForm.tsx:967]: Could, but they just finished creating or editing it. I feel it is enough just to give feedback that it was created.

Comment (showeimer) [reply frontend/src/routes/Credentials/CredentialsForm.tsx:967]: Checked with Joy, she seems to like including the name.  We should probably standardize the toasts across this, cluster pools, and discovered clusters, so maybe:

Title: Credential created
Message: james-credential-1 was successfully created.

File: frontend/public/locales/en/credentials.json [hunk 2]
```diff
@@ -28,6 +28,8 @@
     "credentialsForm.submitButton.save": "Save",
     "credentialsForm.submitButton.saving": "Saving",
     "credentialsForm.cancelButton.label": "Cancel",
+    "credentialsForm.updated.title": "Credentials updated",
+    "credentialsForm.created.title": "Credentials created",
```
Comment (showeimer) [inline frontend/public/locales/en/credentials.json:32]: Shouldn't be plural

Comment (jamestalton) [reply frontend/public/locales/en/credentials.json:32]: A secret might contain multiple credentials...

Comment (jamestalton) [reply frontend/public/locales/en/credentials.json:32]: We should get docs to review across the board the credential vs credentials.
But not in this issue.

File: frontend/src/routes/Credentials/CredentialsForm.tsx [hunk 3]
```diff
@@ -949,10 +951,22 @@ export function CredentialsForm(props: {
...
                     history.push(NavigationPath.credentials)
                 })
             } else {
                 return createResource(stateToData() as IResource).promise.then(() => {
+                    toastContext.addAlert({
+                        title: t('credentialsForm.created.title'),
+                        message: t('credentialsForm.created.message'.replace('{0}', name)),
```
Comment (showeimer) [inline frontend/src/routes/Credentials/CredentialsForm.tsx:966]: This should work:

```
t('credentialsForm.created.message', { name })
```


```
"credentialsForm.created.message": "{{name}} was successfully created.",
```

**Review by showeimer — DISMISSED** [review]

**Pull request merged**